src/main/java/io/logz/benchmarks/elasticsearch/controllers/SearchController.java

```diff
@@ -73,8 +73,9 @@ public void startSearching(int threadNumber) {
                 try {
                     String currSearch = esController.getSearch();
 
+                    String index = esController.getIndexPrefix().isPresent() ? esController.getIndexPrefix().get() + "*" : esController.getIndexName();
```

Comment:
I think this logic is better at ElasticsearchController
To have some method of `getSearchIndex()` that will return the index, or the index prefix with *